draft-ietf-httpbis-cache-latest.xml

```diff
@@ -249,6 +248,15 @@
 </section>
 </section>
 
+<section title="Terminology and Core Concepts" anchor="terminoly">
+<t>
+  See <xref target="Semantics" x:fmt="of" x:rel="#terminology"/> for
+  general terminology and core concepts of HTTP.
+</t>
+</section>
+
+</section>
```

Comment:
Does this still build? It looks like an extra section closure was added.

Reply:
Yes, I changed the nesting of a section.

Reply:
BTW, the anchor on that section has a typo.